features/tasks.feature

```diff
@@ -295,20 +219,14 @@ Feature: Tasks
             "doneBefore":"@string@.isDateTime().startsWith('2018-03-02T12:00:00')",
             "comments":"#bob",
             "updatedAt":"@string@.isDateTime()",
-            "isAssigned":true,
-            "assignedTo":"bob",
```

Comment:
we don't need these in-app ?

Reply:
I did not find any usages of them in the Courier part, only the Dispatch